admin/class-customizer.php

```diff
@@ -0,0 +1,318 @@
+<?php
+/**
+ * @package WPSEO\Admin\Customizer
+ */
+
+/**
+ * Class with functionality to support WP SEO settings in WordPress Customizer.
+ */
+class WPSEO_Customizer {
+
+	/**
+	 * Construct Method.
+	 */
+	public function __construct() {
+		add_action( 'customize_register', array( $this, 'wpseo_customize_register' ) );
+	}
+
+	/**
+	 * Function to support WordPress Customizer
+	 *
+	 * @param WP_Customize_Manager $wp_customize
+	 */
+	public function wpseo_customize_register( $wp_customize ) {
+		$this->breadcrumbs_section( $wp_customize );
```

Comment:
The idea was because it is called from an action that it isn't 'nice' to put `$wp_customize` in an attribute but I don't feel very strongly about this.